src/NUnitFramework/framework/Constraints/ExactCountConstraint.cs

```diff
@@ -31,9 +31,19 @@ namespace NUnit.Framework.Constraints
     /// item in a collection, succeeding only if a specified
     /// number of items succeed.
     /// </summary>
-    public class ExactCountConstraint : PrefixConstraint
+    public class ExactCountConstraint : Constraint
     {
-        private int expectedCount;
+        private readonly int expectedCount;
+        private readonly IConstraint itemConstraint;
```

Comment:
The dotSettings is already showing naming convention inspections as warnings. Somehow my personal settings were overriding the project settings. I'm still not clear what  I messed up but I removed my preference for camelCase and added my preference back and it's working. As far as the experience goes, it's perfect. You see blue squiggles in the code but no warnings in the error list.